h2o-r/tests/testdir_jira/runit_pubdev_4630_r2h2o_large.R

```diff
@@ -0,0 +1,74 @@
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
+source("../../scripts/h2o-r-test-setup.R")
+
+#library(slam)
+#library(data.table)
+
+# In PUBDEV-4630: it was complaint by some users that as.h2o is very slow with spare matrices.  Hence,
+# Peter Ellis suggested a fix for this.  His fix fixed the speed problem but was generating wrong answers.
+# For rows with all zeros, he will replace it another row instead of generating a row of zeros.  I fixed
+# this.  In this test, I will compare as.h2o with his fix and my fix with answers generated by original 
+# as.h2o.  Then, I will randomly generate some sparse matrix and just count the number of ones in the as.h2o frame
+# and the original matrix.
+
+check.as.h2o<- function() {
```

Comment:
I have done this manually and it works.  It generates a warning in this case and the run time will be higher than otherwise.

This can be done by inspection.  I will need to change frame.R to set the options there and this could slow done the operation.